app/views/layouts/application.html.erb

```diff
@@ -1,16 +1,29 @@
-<!DOCTYPE html>
+<%# <!DOCTYPE html>
 <html>
   <head>
     <title>HelloRailsReact</title>
     <meta name="viewport" content="width=device-width,initial-scale=1">
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
 
-    <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
-    <%= javascript_include_tag "application", "data-turbo-track": "reload", defer: true %>
-  </head>
+    <%# <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %> 
+    <%# <%= javascript_include_tag "application", "data-turbo-track": "reload", defer: true %> 
+  <%# </head> %>
 
+  <%# <body> %>
+    <%# <%= yield %> 
+  <%# </body> %>
+<%# </html> %> 
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Hello Rails React</title>
+    <%= javascript_pack_tag 'application' %>
+    <%= stylesheet_pack_tag 'application' %>
+  </head>
   <body>
+    <div id="root"></div>
```

Comment:
- [ ] It would be better if we create a `root route` and `root` view for which React and Redux connects to. Currently, the frontend app is not communicating successfully with the backend API as it should. Let us kindly work on fixing it